src/bp-core/bp-core-update.php

```diff
@@ -636,13 +638,13 @@ function bp_update_to_1_4_0() {
  */
 function bp_update_to_1_4_3() {
 	global $wpdb;
-	$bp = buddypress();
-	$squery = "SELECT GROUP_CONCAT( pm.meta_value ) as media_id FROM {$wpdb->posts} p, {$wpdb->postmeta} pm WHERE p.ID = pm.post_id and p.post_type in ( 'forum', 'topic', 'reply' ) and pm.meta_key = 'bp_media_ids' and pm.meta_value != ''";
+	$bp      = buddypress();
+	$squery  = "SELECT GROUP_CONCAT( pm.meta_value ) as media_id FROM {$wpdb->posts} p, {$wpdb->postmeta} pm WHERE p.ID = pm.post_id and p.post_type in ( 'forum', 'topic', 'reply' ) and pm.meta_key = 'bp_media_ids' and pm.meta_value != ''";
 	$records = $wpdb->get_col( $squery );
 	if ( ! empty( $records ) && bp_is_active( 'media' ) ) {
 		$records = reset( $records );
-		if ( !empty( $records ) ) {
-			$update_query = "UPDATE {$bp->media->table_name} SET `privacy`= 'forums' WHERE id in (" . $records . ")";
+		if ( ! empty( $records ) ) {
+			$update_query = "UPDATE {$bp->media->table_name} SET `privacy`= 'forums' WHERE id in (" . $records . ')';
```

Comment:
@dipskakadiya This will break the query as this will be a 
```suggestion
			$update_query = "UPDATE {$bp->media->table_name} SET `privacy`= 'forums' WHERE id in (" . $records . ")";
```